qemu/tests/numa_dist.py

```diff
@@ -0,0 +1,48 @@
+import ast
+import logging
+
+from virttest import error_context
+from virttest.utils_misc import NumaInfo
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Simple test to check if NUMA dist options are being parsed properly
+    """
+    vm = env.get_vm(params["main_vm"])
+    expected_numa_dist = {}
+    guest_numa_nodes = params.objects("guest_numa_nodes")
+    for numa_node in guest_numa_nodes:
+        numa_node_dist_value = ['unset' for i in range(len(guest_numa_nodes))]
+        numa_params = params.object_params(numa_node)
+        numa_nodeid = numa_params["numa_nodeid"]
+        numa_dist = ast.literal_eval(numa_params.get("numa_dist", "[]"))
+        for dist in numa_dist:
+            dst_node = dist[0]
+            distance_value = dist[1]
+            numa_node_dist_value[dst_node] = str(distance_value)
+        expected_numa_dist[int(numa_nodeid)] = numa_node_dist_value
+
+    for src_id, dist_info in expected_numa_dist.items():
+        # The distance from a node to itself is always 10
+        dist_info[src_id] = '10'
```

Comment:
It should be an int here?

Reply:
No, it must be a string since the info get from guest os is a string: `numa_info_guest.distances`.

Reply:
I see.